src/config.js

```diff
@@ -1,16 +1,38 @@
 import os from 'os';
 import path from 'path';
 
-import importFresh from 'import-fresh';
 import camelCase from 'camelcase';
 import decamelize from 'decamelize';
+import { fs } from 'mz';
+import parseJSON from 'parse-json';
 
 import fileExists from './util/file-exists.js';
 import { createLogger } from './util/logger.js';
 import { UsageError, WebExtError } from './errors.js';
 
 const log = createLogger(import.meta.url);
 
+export const WARN_LEGACY_JS_EXT = [
+  'should be renamed to .cjs or .mjs to ensure its format is not ambiguous.',
```

Comment:
Maybe add a comment to indicate that this message is added to another string when it is printed. Other strings below are full sentences but this one is partial.